lib/modules/manager/buildpacks/index.ts

```diff
@@ -0,0 +1,12 @@
+import type { Category } from '../../../constants';
+import { DockerDatasource } from '../../datasource/docker';
+export { extractPackageFile } from './extract';
+
+export const defaultConfig = {
+  commitMessageTopic: 'buildpacks project descriptor {{depName}}',
```

Comment:
This is a pretty long string compared to other managers. Are you certain that it needs to be so long to be clear? e.g. we normally use "dependency {{depName}}" or "image {{depName}}", "crate {{depName}}", etc.

Reply:
> Are you certain that it needs to be so long to be clear?

I agree that it doesn't have to be that long, would it be ok to change to `buildpack {{depName}}`?

Reply:
Yes that would be better

Reply:
done